certificate-policy.md

```diff
@@ -841,289 +811,247 @@ All persons filling trusted roles shall be selected on the basis of loyalty, tru
+- CA accreditation 
 - Certificate policy
 - Certification practice statement
 - Contractual obligations and other agreements concerning operations of the CA
 - System and equipment configuration
--	Modifications and updates to system or configuration
--	Certificate requests
--	All certificates issued and/or published
--	Revocation requests
--	Subscriber identity authentication data
--	Subscriber agreements
--	Documentation of receipt of tokens
--	All CRLs issued and/or published
--	Other data or applications to verify archive contents
--	Compliance Auditor reports
--	Any changes to the Audit parameters, e.g. audit frequency, type of event audited
--	Any attempt to delete or modify the Audit logs
--	Whenever the CA generates a key (Not mandatory for single session or one-time use symmetric keys)
--	All changes to the trusted public keys, including additions and deletions
--	The export of private and secret keys (keys used for a single session or message are excluded)
--	The approval or rejection of a certificate status change request
--	Appointment of an individual to a Trusted Role
--	Destruction of cryptographic modules
--	All certificate compromise notifications
--	Remedial action taken as a result of violations of physical security
--	Violations of Certificate Policy
--	Violations of Certification Practice Statement
+  -	Modifications and updates to system or configuration
+- Certificate requests
+- All certificates issued 
+- Revocation requests
+- Subscriber identity authentication data
+- Subscriber agreements
+- Documentation of receipt of tokens
+- All CRLs issued 
+- Qualified Auditor reports
+- Any changes to the Audit parameters, e.g. audit frequency, type of event audited
+- Any attempt to delete or modify the Audit logs
+- Whenever the CA generates a key (Not mandatory for single session or one-time use symmetric keys)
+- All changes to the trusted public keys, including additions and deletions
+- The export of CA private and secret keys (keys used for a single session or message are excluded)
+- The approval or rejection of a certificate status change request
+- Appointment of an individual to a Trusted Role
+- Other data or applications to verify archive contents
+- Destruction of cryptographic modules
+- All certificate compromise notifications
+- Violations of Certificate Policy
+- Violations of Certification Practice Statement
 
 ### 5.5.2 Retention period for archive
-The CA shall retain all documentation relating to certificate requests and the verification thereof, and all Certificates and revocation thereof, for a minimum of 10 years and 6 months without any loss of data after any Certificate based on that documentation ceases to be valid.
+The CA shall retain all documentation relating to certificate requests and the verification thereof, and all Certificates and revocation information thereof, for a minimum of seven years without any loss of data after any Certificate based on that documentation ceases to be valid.
```

Comment:
Is 7 years meant to reference current NARA rules? We may have an interest in keeping some data beyond 7 years, and referencing records schedules timelines implicitly emphasizes that deletion is default-encouraged/okay after that timeline. I think we would benefit from having a sentence here that says that after 7 years, any data deletion should still be approved by FPKIPA.

Reply:
The update was to address #257 and inconsistencies identified by Scott Perry.  

-The 7 years is Baseline Requirements.  
-The original 10 years, 6 months was Federal Common Policy (or Federal Bridge Certificate Policy).  
-The 10 years, 6 months was rooted (pun intended) in 800-63 + Digital / Electronic Signatures + CFRs.  
- One archival reference: https://www.archives.gov/records-mgmt/policy/pki.html#4-3

The 10 years, 6 months wasn't applicable to the purpose of this infrastructure - so I reverted to the baseline requirements.  

I'm hesitant to include that any data deletion after 7 years requires approval by the PA, as a voting body, or directly by the PA co-chairs.  I think it is better to state the basic requirements, and allow any new NARA requirements or FISMA requirements to take precedence (and not require a policy to be changed or a vote / approval to be taken).  

Thoughts? We can open an issue to discuss but I want to merge!